google-cloud-firestore/lib/google/cloud/firestore/service.rb

```diff
@@ -194,6 +196,18 @@ def document_mask mask
 
           Google::Cloud::Firestore::V1::DocumentMask.new field_paths: mask
         end
+
+        def read_time_to_timestamp time
+          return nil if time.nil?
+
+          # Force the object to be a Time object.
+          time = time.to_time.utc
```

Comment:
What is this doing? What type could `time` be, where this kind of conversion would be needed?

I'm asking because all the `read_time` parameters above are documented to require type `Time`. This could be problematic by itself because users might want to pass a DateTime, or maybe even a Numeric (representing seconds since the posix Epoch). However, if you require `Time`, then it seems like this line wouldn't be needed. If this line (or something like it) is needed, and you're supporting some other classes, then those should be called out in the `@param` documentation.